README.md

```diff
@@ -63,84 +68,80 @@ ground = Entity(
+2. Insert the following code into your newly created file:
+
+```python
+from ursina import *
 
 app = Ursina()
 
 player = Entity(
-    model = 'cube' ,           # finds a 3d model by name
+    model = 'cube',
     color = color.orange,
     scale_y = 2
-    )
+)
 
-def update():                  # update gets automatically called by the engine.
+def update(): 
     player.x += held_keys['d'] * .1
     player.x -= held_keys['a'] * .1
 
-
-app.run()                     # opens a window and starts the game.
+app.run()
 ```
 
-3) Type this in the terminal to start the game:
+3. Run the game by typing the following command in your terminal:
 
-       python ursina_game.py
-   If you use Atom, I recommend installing the package atom-python-run to run your scripts with the press of a button.
+```bash
+python ursina_game.py
+```
 
-4) You can now move the orange box around with 'a' and 'd'!
+Note: If you're using Atom, we recommend installing the package atom-python-run to execute your scripts at the press of a button.
 
-   To close the window, you can by default, press shift+q or press the red x. to disable this, write `window.exit_button.enabled = False` somewhere in your code.
+4. Now, you can move the orange box around with 'a' and 'd' keys!
 
+To exit the game, press shift+q or click on the red x in the window's corner. To disable this functionality, insert `window.exit_button.enabled = False` into your code.
 
-## Project Structure
-```
 ## Project Structure
 
+```
 📁docs
     📃index.txt
     📃documentation.txt
     📃inventory_tutorial.txt
     ...
-        # text files for the website. gets turned into .html files with sswg.
-
     📃cheat_sheet.html
-        # auto generated documentation made with documentation_generator.py.
-
     📃tutorial_generator.py
-        # turns specific .py files into .txt files, which can then be turned into .html by sswg.
-        # this extracts the comments from the source files into description for that step and the code after into code blocks.
-        # see platformer_tutorial.py for an example.
 
-📁samples               # small example games.
-
-📁ursina                # the actual ursina module.
-    📁audio                 # built-in audio clips.
-    📁editor                # the 3d level editor for ursina.
-    📁fonts                 # built-in fonts.
-    📁models                # .blend files, source files, for built-in 3d models.
-        📁procedural            # classes for generating 3d models, like Cylinder, Quad and Terrain.
-    📁models_compressed     # .blend files converted to .ursinamesh.
-    📁prefabs               # higher level classes like Draggable, Slider, Sprite, etc.
+📁samples
+📁ursina
+    📁audio
+    📁editor
+    📁fonts
+    📁models
+        📁procedural
+    📁models_compressed
+    📁prefabs
 
     📃__init__.py
     📃application.py
     📃audio.py
     ...
-        # ursina base modules, like code for Entity, input_handler, Text, window and so on.
-
 ```

```

Comment:
I don't think it's a good idea to remove the descriptions here. Even if there's some on the footer, it's better to read text linearly than in a non-linear way.